target/riscv/cpu_bits.h

```diff
@@ -495,6 +495,7 @@
 #define RISCV_EXCP_INST_PAGE_FAULT         0xc /* since: priv-1.10.0 */
 #define RISCV_EXCP_LOAD_PAGE_FAULT         0xd /* since: priv-1.10.0 */
 #define RISCV_EXCP_STORE_PAGE_FAULT        0xf /* since: priv-1.10.0 */
+#define RISCV_POLICY_VIOLATION_FAULT       0x11 /* RESERVED */
```

Comment:
Might be better to use 24–31 or 48–63, since those exception codes are available for custom use.